dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -427,18 +423,21 @@ export const CommentLayout = ({ CAPI, NAV, format }: Props): JSX.Element => {
 										showAvatar && minHeightWithAvatar,
 									]}
 								>
-									{/* TOP - we use divs here to position content in groups using flex */}
-									<div css={!showAvatar && headlinePadding}>
-										<ArticleHeadline
-											format={format}
-											headlineString={CAPI.headline}
-											tags={CAPI.tags}
-											byline={CAPI.author.byline}
-											webPublicationDateDeprecated={
-												CAPI.webPublicationDateDeprecated
-											}
-										/>
-									</div>
+									{/* TOP - we position content in groups here using flex */}
+									<ArticleHeadline
+										format={format}
```

Comment:
From the perspective of a layout file, I find this refactored component much easier to reason about - I think it'll be more intuitive to add a correctly padded ArticleHeadline into a new layout file too.